driver_scripts/driver_grid.hera.sh

```diff
@@ -88,6 +88,7 @@ fi
 export home_dir=$SLURM_SUBMIT_DIR/..
 export TMPDIR=/scratch2/NCEPDEV/stmp1/$LOGNAME/fv3_grid.$gtype
 export out_dir=/scratch2/NCEPDEV/stmp1/$LOGNAME/C${res}
+export add_lake=false
 
```

Comment:
I would add some comments in the prolog about how/when to set this flag.  That would be helpful for the user.

Reply:
How about adding this line?

# add_lake=T will add lake fraction/depth to oro_data and adjust land_frac/slmsk accordingly
export add_lake=false

Reply:
That would be ok.  I would move the comment, export of add_lake, and the associated error handling before the export of home_dir.